frog/imports/ui/GraphEditor/SidePanel/ExpandButton.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import { connect } from '../store';
+
+export default connect(({
+  store: { ui: { setSidepanelOpen, sidepanelOpen } }
+}) => {
+  if (sidepanelOpen) {
+    return null;
+  }
+  return (
+    <CloseDiv>
+      <i className="fa fa-ellipsis-v" onClick={() => setSidepanelOpen(true)} />
```

Comment:
There is an eslint error here, you can ignore it if you want.
```
/.../FROG/frog/imports/ui/GraphEditor/SidePanel/ExpandButton.js
  14:7  error  Visible, non-interactive elements should not have mouse or keyboard event listeners  jsx-a11y/no-static-element-interactions
```

My codefresh  pipeline, that I reopened it because I can't see the builds otherwise, sent a green light. Let's hope they have better collaborative support soon. I fixed my pipeline to run the tests now, normally).